dataland-frontend/src/components/pages/SingleDataRequest.vue

```diff
@@ -259,6 +260,17 @@ export default defineComponent({
         this.submitted = true;
       }
     },
+    /**
+     * Populates the availableFrameworks property in the format expected by the dropdown filter
+     */
+    retrieveFrameworkOptions() {
+      this.frameworkOptions = ARRAY_OF_FRAMEWORKS_WITH_VIEW_PAGE.map((dataTypeEnum) => {
+        return {
+          value: dataTypeEnum,
+          label: humanizeStringOrNumber(dataTypeEnum),
```

Comment:
Since this is a separate pull request, do we need to test this? Could e.g. be done by explicitly searching for "WWF Pathway to Paris" inside the drop-down (which was shown as "p2p" before). Or is this not necessary, as it is only a styling issue?

Reply:
Yes wo should, if we do not yet. Let me check that qickly.

Reply:
expanded the test